src/Routes/WizardPage/WizardPage.tsx

```diff
@@ -43,6 +43,26 @@ const WizardPage = () => {
   const linkLearnMoreAbout = 'https://access.redhat.com/articles/1586893';
   const linkLearnMoreAboutRemovingDirectoryAndDomainServices = 'https://access.redhat.com/articles/1586893';
 
+  useEffect(() => {
```

Comment:
I think the code can be bit simplified. So for inspiration:

```ts
  if (appContext.rbac.isLoading) {
    return; // we could return centrally aligned spinner here
  }

  useEffect(() => {
    if (!appContext.rbac.permissions.hasTokenCreate || !appContext.rbac.permissions.hasDomainsUpdate) {
      console.debug('wizard: missing permissions to create token or update domains');
      navigate('/no-permissions', { replace: true });
    }
  }, [appContext.rbac.permissions]);
```

Up to you what to use.